ddl/ddl_worker.go

```diff
@@ -38,20 +38,20 @@ var RunWorker = true
 type workerType byte
 
 const (
-	// generalWorker is the worker who handles all DDL worker now.
-	// TODO: update the comments when we support the addIdxWorker.
+	// generalWorker is the worker who handles all DDL statements except “add index”.
 	generalWorker workerType = 0
-	addIdxWorker  workerType = 1
+	// addIdxWorker is the worker who handles the operation of adding indexes.
+	addIdxWorker workerType = 1
 )
 
 // worker is used for handling DDL jobs.
-// Now we have two kinds of workers, but we only use the generalWorker.
-// TODO: update the comments when we support the addIdxWorker.
+// Now we have two kinds of workers.
 type worker struct {
-	id     int
-	tp     workerType
-	quitCh chan struct{}
-	wg     sync.WaitGroup
+	id       int
+	tp       workerType
+	ddlJobCh chan struct{}
```

Comment:
What is this used for?

Reply:
`ddlJobCh` used to be member of ddl, but now, we have 2 kinds of workers,  we need two `ddlJobCh` for every kind of worker. So move `ddlJobCh` from `ddl` to `worker`